packages/app/prisma-utils/biome.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "../../../node_modules/@biomejs/biome/configuration_schema.json",
+  "extends": ["../../../node_modules/@lokalise/biome-config/configs/biome-base.jsonc"]
```

Comment:
@kibertoad I guess on package we want to have our biome base config right?

Reply:
yup!